### PR TITLE
Group Comm MOCK Refactor

### PR DESCRIPTION
**Pull request by eilinora**
Moving MOCK's and app state setup into subfolders to make management easier

### CODE REVIEW

File: src/group/communication/api.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            "flag_spam"
+         ]
+      }
+   }
+};
+
+export const MOCK_GROUP_COMMUNICATION_FOLLOWERS = [
+	MOCK_CHAPTER_MEMBER,
+	MOCK_CHAPTER_MEMBER,
+	MOCK_CHAPTER_MEMBER
```
Comment (mmcgahan) [inline src/group/communication/api.js]: Probably worth making a copy of each of these and changing the id, which is sometimes used as an assumed-unique value, e.g.

```js
[
  { ...MOCK_CHAPTER_MEMBER, id: 12345 },
  { ...MOCK_CHAPTER_MEMBER, id: 23456 },
]
```

**Comment by eilinora** [comment]
uck forgot to push that up before, had to do that for the tests to work right :)

**Comment by sadafie** [comment]
I don't know if anyone has a strong preference, but I personally find it difficult to work in a repo that has several files of the same name `app.js` `api.js`. One alternative idea would just be to do `src/group/communication.js` and have that be all the mocks needed, regardless of being an "api" or "app" mock.

**Comment by eilinora** [comment]
I'm a fan of the app/api files (obviously) only cause it's keeps the MOCK's organized similar to the root mocks. But I could see the logic in a `communication.js` file. 

Does anyone else have a particular preference? This branch is a blocker to the MUP PR's passing tests.

**Comment by sadafie** [comment]
I'd be interested in getting more thoughts, but also this shouldn't be a blocker, so feel free to merge and we could always change as a follow up if we decide to.

**Comment by eilinora** [comment]
👍

**Comment by mmcgahan** [comment]
another simple way to address the concern is to rename `src/group/communication/api.js` to `src/group/communication/apiGroupComm.js`

a less-simple way to do it (that I think would be preferable?) is to make a `src/api/` directory that would have separate modules like `src/api/communication.js`

**Comment by sadafie** [comment]
>a less-simple way to do it (that I think would be preferable?) is to make a src/api/ directory that would have separate modules like src/api/communication.js

+1 on this

**Pull request merged**